Code/Framework/AzCore/AzCore/UnitTest/TestTypes.h

```diff
@@ -85,6 +87,24 @@ namespace UnitTest
         ~ScopedAllocatorSetupFixture() { TeardownAllocator(); }
     };
 
+    // Like ScopedAllocatorSetupFixture, but test/benchmark agnostic
```

Comment:
can you make this the base class of the one above? That way we deduplicate the code

Reply:
Done